### DISCUSSION TITLE
weird bug in specific safari on newish m1 mac

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#583] [! Welcome - readme first!]
I've been sent a bug from my client where this comp doesn't play correctly in this specific version of safari on what looks like a new machine.... 

![image002 (1)](https://github.com/cables-gl/cables_docs/assets/36418408/55324f10-a0cb-4953-b0a0-0b7d535d6695)

<img width="683" alt="Screenshot 2023-10-03 at 08 53 08" src="https://github.com/cables-gl/cables_docs/assets/36418408/bc4c24ff-e78d-4717-8def-89d03f2f950a">

Not sure why...

https://designmiami.com/
https://cables.gl/edit/qUEGtE

**Comment by steam0r** [comment]
this somehow seems to be an issue with ammo, that we cannot seem to fix :|
